bucket/DIE.json

```
@@ -0,0 +1,47 @@
+{
+    "description": "Detect It Easy, or abbreviated \"DIE\" is a program for determining types of files.",
+    "homepage": "https://ntinfo.biz/index.html",
+    "license": "MIT",
+    "version": "2.05",
+    "url": "https://github.com/horsicq/DIE-engine/releases/download/2.05/die_win32_portable_2.05.zip",
+    "hash": "c597381d73a7ce6ad1acc216c5ddd9010cde56c830063eaeee9411974f60963e",
+    "extract_dir": "die_win32_portable/base",
+    "installer": {
+        "script": [
+            "if (Test-Path \"$persist_dir\\die.ini\") { Copy-Item \"$persist_dir\\die.ini\" \"$dir\" -Force }",
+            "if (Test-Path \"$persist_dir\\diec.ini\") { Copy-Item \"$persist_dir\\diec.ini\" \"$dir\" -Force }"
+        ]
+    },
```

Why force is needed??

```suggestion
    "pre_install": "Copy-Item \"$persist_dir\\*\" \"$dir\" -Include 'die.ini', 'diec.ini' -ErrorAction SilenltyContinue",
```